src/main/java/core/basesyntax/FileReaderImpl.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String filePath) throws IOException {
```

Comment:
Make sure to handle the potential `IOException` more gracefully. Rethrowing it as a RuntimeException with an informative message would be more appropriate, as per the checklist guidelines.